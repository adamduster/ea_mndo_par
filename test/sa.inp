/home/aduster/apps/mndo/mndo99
/tmp
hofweight 10.00000
bondweight 100.00000
angweight 5.00000
dihweight 0.1111111
seed 262016
nGen 20
nPop 7
parameters
  1  1  1      -12.45828647   0.4
  3  1  1        1.25906452   5
  5  1  1       -3.40077019   10
 11  1  1        0.06957348   25
 16  1  1        0.25412774   50
 18  1  1        0.35603700   50
 91  1  1       12.84800000   0.5
  1  6  1      -50.55997310   0.1  
  2  6  1      -39.60463506   0.1  
  3  6  1        1.27811536   10   
  5  6  1       -7.15025194   5      
  6  6  1       -4.00957578   5    
  8  6  1       -5.63953826   5    
  9  6  1       -6.19878821   5    
 10  6  1       -4.23218141   5    
 11  6  1        0.09252392   50   
 12  6  1        0.05324243   30   
 13  6  1        0.09913670   30   
 14  6  1        0.10032178   30   
 15  6  1        0.05514590   30   
 16  6  1        0.41166670   50   
 18  6  1        0.10387553   50   
 91  6  1       12.23000000   0.75 
 92  6  1       11.08000000   0.75 
 93  6  1       11.47000000   0.75 
 94  6  1        9.84000000   0.75 
 95  6  1        2.43000000   0.75 
  1  8  1     -105.79319826   0.1
  2  8  1      -78.90502490   0.1
  3  8  1        1.20838191   10
  5  8  1      -14.42831206   5
  6  8  1       -8.77115999   5
  8  8  1      -12.94993030   5
  9  8  1      -13.56404773   5
 10  8  1       -9.42205182   5
 11  8  1        0.12951559   50
 12  8  1        0.09280489   30
 13  8  1        0.16102284   30
 14  8  1        0.14523894   30
 15  8  1        0.10988763   30
 16  8  1        0.55290848   50
 18  8  1        0.06227392   50
 91  8  1       15.42000000   0.75
 92  8  1       14.52000000   0.75 
 93  8  1       14.48000000   0.75 
 94  8  1       12.98000000   0.75 
 95  8  1        3.94000000   0.75 
