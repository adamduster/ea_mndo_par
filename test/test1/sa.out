 /home/aduster/apps/mndo/mndo99                                                
 Hof Weight:   10.0000000000000     
 Bond Weight:    100.000000000000     
 Angle Weight:    5.00000000000000     
 Dihedral Weight:   0.111111100000000     
 Population Size:                     12
 Number of Generations:                  10000
 The input parameters:
TYP EL OPT         PAR VALUE   PERCENT VARIED
  1  1   1    -12.4582864700    0.40%
  3  1   1      1.2590645200    5.00%
  5  1   1     -3.4007701900   10.00%
 11  1   1      0.0695734800   25.00%
 16  1   1      0.2541277400   50.00%
 18  1   1      0.3560370000   50.00%
 91  1   1     12.8480000000    0.50%
  1  6   1    -50.5599731000    0.10%
  2  6   1    -39.6046350600    0.10%
  3  6   1      1.2781153600   10.00%
  5  6   1     -7.1502519400    5.00%
  6  6   1     -4.0095757800    5.00%
  8  6   1     -5.6395382600    5.00%
  9  6   1     -6.1987882100    5.00%
 10  6   1     -4.2321814100    5.00%
 11  6   1      0.0925239200   50.00%
 12  6   1      0.0532424300   30.00%
 13  6   1      0.0991367000   30.00%
 14  6   1      0.1003217800   30.00%
 15  6   1      0.0551459000   30.00%
 16  6   1      0.4116667000   50.00%
 18  6   1      0.1038755300   50.00%
 91  6   1     12.2300000000    0.75%
 92  6   1     11.0800000000    0.75%
 93  6   1     11.4700000000    0.75%
 94  6   1      9.8400000000    0.75%
 95  6   1      2.4300000000    0.75%
  1  8   1   -105.7931982600    0.10%
  2  8   1    -78.9050249000    0.10%
  3  8   1      1.2083819100   10.00%
  5  8   1    -14.4283120600    5.00%
  6  8   1     -8.7711599900    5.00%
  8  8   1    -12.9499303000    5.00%
  9  8   1    -13.5640477300    5.00%
 10  8   1     -9.4220518200    5.00%
 11  8   1      0.1295155900   50.00%
 12  8   1      0.0928048900   30.00%
 13  8   1      0.1610228400   30.00%
 14  8   1      0.1452389400   30.00%
 15  8   1      0.1098876300   30.00%
 16  8   1      0.5529084800   50.00%
 18  8   1      0.0622739200   50.00%
 91  8   1     15.4200000000    0.75%
 92  8   1     14.5200000000    0.75%
 93  8   1     14.4800000000    0.75%
 94  8   1     12.9800000000    0.75%
 95  8   1      3.9400000000    0.75%

--------------------------------------------------------------------------------

